Content/default/src/Client/Index.fs

```diff
@@ -15,29 +13,33 @@ type Msg =
     | AddedTodo of Todo
 
 let todosApi =
-    Remoting.createApi()
+    Remoting.createApi ()
     |> Remoting.withRouteBuilder Route.builder
     |> Remoting.buildProxy<ITodosApi>
 
-let init(): Model * Cmd<Msg> =
-    let model =
-        { Todos = []
-          Input = "" }
-    let cmd = Cmd.OfAsync.perform todosApi.getTodos () GotTodos
+let init () : Model * Cmd<Msg> =
+    let model = { Todos = []; Input = "" }
+
+    let cmd =
+        Cmd.OfAsync.perform todosApi.getTodos () GotTodos
+
     model, cmd
 
-let update (msg: Msg) (model: Model): Model * Cmd<Msg> =
+let update (msg : Msg) (model : Model) : Model * Cmd<Msg> =
     match msg with
-    | GotTodos todos ->
-        { model with Todos = todos }, Cmd.none
-    | SetInput value ->
-        { model with Input = value }, Cmd.none
+    | GotTodos todos -> { model with Todos = todos }, Cmd.none
+    | SetInput value -> { model with Input = value }, Cmd.none
     | AddTodo ->
         let todo = Todo.create model.Input
-        let cmd = Cmd.OfAsync.perform todosApi.addTodo todo AddedTodo
+
+        let cmd =
+            Cmd.OfAsync.perform todosApi.addTodo todo AddedTodo
```

Comment:
Just curious, surely the line wasn't considered too long?

Reply:
That part is controlled by the max_line_length setting.

Reply:
And the default now is what? 120? 80?

Reply:
Default is 120, [see default settings](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#configuration).

Reply:
Oh, there are two things in play here:
* the max_line_length will determine if the function application should remain in one line or go multiline:
```fsharp
Cmd.OfAsync.perform 
    todosApi.addTodo 
    todo 
    AddedTodo
```
* setting [fsharp_max_value_binding_width](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#fsharp_max_value_binding_width) controls wether the expression after the `=` sign remains on the same line. (Threshold is 40 there)